pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+env =
+    DJANGO_SETTINGS_MODULE=config.settings.test
+    DJANGO_DOT_ENV_FILE=.env.test
```

Comment:
Is this required?

Reply:
No, I realised tests where not picking the environment variables and got into this PR